src/status_im/contexts/shell/activity_center/notification/dapp_connection/view.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.shell.activity-center.notification.dapp-connection.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [status-im.contexts.shell.activity-center.notification.common.view :as common]
+    [utils.datetime :as datetime]
+    [utils.i18n :as i18n]
+    utils.url))
+
+(defn view
+  [{:keys [customization-color active-swipeable notification extra-fn]}]
+  (let [{:keys [timestamp read dapp-url dapp-icon-url]} notification]
+    [common/swipeable
+     {:left-button      common/swipe-button-read-or-unread
+      :left-on-press    common/swipe-on-press-toggle-read
+      :right-button     common/swipe-button-delete
+      :right-on-press   common/swipe-on-press-delete
+      :active-swipeable active-swipeable
```

Comment:
The `active-swipeable` prop is no longer used because the `common/swipeable` component will get this ref from the React context.

https://github.com/status-im/status-mobile/blob/8c2d5398b0add4906c397acfee2c91307e41a557/src/status_im/contexts/shell/activity_center/view.cljs#L78

Reply:
thanks, removed